Specs/Number/English/NumberModel.json

```diff
@@ -1698,4 +1698,30 @@
     }
   ]
 }
+,
+{
+  "Input": "1 234 567",
```

Comment:
Looks good. But can you add a few more full sentence examples for English?
Also, cases with regular spaces and cases with narrow-space.
Thanks.